tools/tidy/src/style/NoOldAlwaysSyntax.cpp

```diff
@@ -0,0 +1,58 @@
+//! @file NoOldAlwaysSyntax.h
```

Comment:
Looks like some of these files are missing the top comment line, and also have the wrong file name listed. You don't need the doxygen @file style comments here either for source files since nothing in them needs to be documented.